README.md

```diff
@@ -1,15 +1,17 @@
 # donation-tracker-toplevel
 
+## AS OF 2019/11/01 PYTHON 3.6/3.7 IS REQUIRED (3.8 IS UNTESTED)
+
 Contains the settings and configuration for creating a simple deploy of the tracker.
 
 In order to deploy the tracker, some boilerplate code is neccessary for configuration and management. The goal of this repository is to make doing so as simple as possible for any given user to get started developing on the tracker.
 
 ## Getting a Working Copy of the Tracker
 
 1. [Install Git](http://www.git-scm.com/download). I'm assuming if you're here, you know enough about git and version control to get started. You can check if you have git with the command `which git`, and which version you have with `git --version`.
-1. [Install Python](https://www.python.org/downloads/). Currently, the tracker only supports version 2.7 for now. Thus, getting whatever the latest 2.7 version is ideal. You can determine if Python is installed with the command `which python`, and which version of Python you have with the command `python -V`. Python version 3.x is **not** supported by our code at this time. If someone wants that to change, they will have to do the legwork themselves, we are not in a position to support both versions at this time.
+1. [Install Python](https://www.python.org/downloads/). The code has been tested with 3.6 and 3.7. 3.5 will not work, and 3.8 is not thoroughly tested. You can determine if Python is installed with the command `which python`, and which version of Python you have with the command `python -V`.
```

Comment:
I don't think mentioning 3.5 is necessary. `Requires at least 3.6 (though 3.8 has not been fully tested)` seems like it would cover everything necessary.